main.py

```diff
@@ -6,6 +6,7 @@
 cfg = {
     ##########################################################################
     "process_to_switch": "prime95",  # name of process (e.g. "cinebench")    #
+    "starting_core": 1, # Core to start iteration                            #
```

Comment:
I like how you let the user specify a core, and not a thread. The core is more relevant to the user than the thread.
You have prompted me to change the rest of the config to also be based on core numbers and not thread numbers. And I've discovered a bug with hyper_threading off. Thanks!